### DISCUSSION TITLE
Activity Detector

### DISCUSSION THREAD

**Discussion by Blakedows2001** [ScratchAddons/ScratchAddons#4937] [Ideas]
This addon can let you know when a Scratcher is active, semi-active, or inactive.
Concepts/Mockups:
![Screenshot 2022-08-01 172649](https://user-images.githubusercontent.com/89300677/182250128-f8dcc308-a9ee-4800-aefe-d824a5a702f0.png)
![Screenshot 2022-08-01 2](https://user-images.githubusercontent.com/89300677/182250151-5a609fc3-c7d6-4842-b241-2cd844559671.png)
![Screenshot 2022-08-01 3](https://user-images.githubusercontent.com/89300677/182250169-44d89626-366e-4c15-9023-add9b457d431.png)
This addon also applies to comments and forum posts.

**Comment by Norbiros** [comment]
See #504 #342 #4484 https://github.com/ScratchAddons/ScratchAddons/issues/4058
